include/zenoh_commons.h

```diff
@@ -781,6 +784,9 @@ typedef struct z_liveliness_token_options_t {
  */
 #if defined(Z_FEATURE_UNSTABLE_API)
 typedef struct z_liveliness_get_options_t {
+  /**
+   * Set query timeout in milliseconds.
```

Comment:
I would suggest to replace with " The timeout for the liveliness query in milliseconds. 0 means default query timeout from zenoh configuration." similarly to how it is formulated for z_get_options.